src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Avatar, Button, Dropdown, Form, Input, Layout, Menu, Space } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+
+import routes from 'constants/routes';
+import { useAuth } from 'hooks/useAuth';
+
+import './styles.scss';
+import snippetslogo from 'assets/img/logoblack.svg';
+
+const { Header } = Layout;
+
+function NavBar() {
+  const { logout, currentUser } = useAuth();
+  const history = useHistory();
+
+  async function handleLogout() {
+    try {
+      await logout();
+      history.push(routes.LOGIN);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error();

```

Comment:
where is the value for the log? 

i think its better if we use antd `Modal.error()` here instead of logging. like this 
![image](https://user-images.githubusercontent.com/47238658/127613764-b371bc5b-1289-4837-a062-116842bed038.png)

Reply:
hi kuya! its for logging out unta so i think no need for modal na.